### ISSUE TITLE
How to get rid of the white border?

### ISSUE THREAD

**Issue by OnkelTem** [carbon-app/carbon#1367]
When make Copy a snippet as an image or save it as PNG I get thick white border around the picture:

![carbon](https://user-images.githubusercontent.com/114060/164985124-a3df5f4d-c9bf-46cc-88f0-a0b06230f305.png)

It doesn't look good on Twitter:

![image](https://user-images.githubusercontent.com/114060/164985302-19290658-8706-46a4-bf2b-06e759207be9.png)

I ran through all the settings but haven't found how to get rid of it. Any ideas? 

My config was:

```json
{
  "paddingVertical": "0px",
  "paddingHorizontal": "0px",
  "backgroundImage": null,
  "backgroundImageSelection": null,
  "backgroundMode": "color",
  "backgroundColor": "rgba(74,144,226,1)",
  "dropShadow": false,
  "dropShadowOffsetY": "20px",
  "dropShadowBlurRadius": "68px",
  "theme": "material",
  "windowTheme": "bw",
  "language": "auto",
  "fontFamily": "Fira Code",
  "fontSize": "12.5px",
  "lineHeight": "134%",
  "windowControls": false,
  "widthAdjustment": false,
  "lineNumbers": false,
  "firstLineNumber": 1,
  "exportSize": "1x",
  "watermark": false,
  "squaredImage": false,
  "hiddenCharacters": false,
  "name": "",
  "width": "600"
}
```

**Comment by mfix22** [comment]
@OnkelTem please share a link to a Carbon image that reproduces this issue.

**Comment by OnkelTem** [comment]
Sorry for the delay.

I found out the reason. The white border is the style applied when you select the 3rd variant in the Window Controls menu:

![image](https://user-images.githubusercontent.com/114060/169711361-7510cc1a-528b-4801-a373-ee6a8ab5a502.png)

And as Twitter applies round borders on images, the original white border get croppe the way I showed above.

So it works as designed. Just avoid using that variant for Twitter.
